src/scss/generic/_heading.scss

```diff
@@ -151,6 +151,16 @@ h6,
     font-size: 12px;
   }
 
+  &--mini {
```

Comment:
This modifier seems to be on a wrong location. Is it necessary to be here or can we move this to inside of card scope?